adapter/sql/util.go

```diff
@@ -51,6 +63,19 @@ func Escape(config Config, field string) string {
 	return escapedField.(string)
 }
 
+func isAliasField(field string) (*aliasField, bool) {
+	if !strings.Contains(strings.ToLower(field), " as ") {
+		return nil, false
+	}
+
+	f := strings.Fields(field)
+	if len(f) != 3 {
```

Comment:
if we decide to use index, maybe we can drop this validation an let whatever happens haha
since we only want to know the first ` as ` word that appears inside the select statement, other case will be resulting in query error and need to be handled without auto escaping